src/DurableTask.AzureStorage/AnalyticsEventSource.cs

```diff
@@ -910,9 +910,9 @@ public void GeneralError(string Account, string TaskHub, string Details, string
         }
 
         [Event(EventIds.GeneralWarning, Level = EventLevel.Warning, Version = 2)]
-        public void GeneralWarning(string Account, string TaskHub, string Details, string AppName, string ExtensionVersion)
+        public void GeneralWarning(string Account, string TaskHub, string Details, string AppName, string ExtensionVersion, string InstanceId)
         {
-            this.WriteEvent(EventIds.GeneralWarning, Account, TaskHub, Details, AppName, ExtensionVersion);
+            this.WriteEvent(EventIds.GeneralWarning, Account, TaskHub, Details, AppName, ExtensionVersion, InstanceId);
```

Comment:
Maybe just for defense-in-depth:
```suggestion
            this.WriteEvent(EventIds.GeneralWarning, Account, TaskHub, Details, AppName, ExtensionVersion, InstanceId ?? string.Empty);
```